apis/mq/v1alpha1/zz_types.go

```diff
@@ -0,0 +1,243 @@
+
+	Revision *int64 `json:"revision,omitempty"`
+}
+
+type ConfigurationRevision struct {
+	Created *metav1.Time `json:"created,omitempty"`
+
+	Description *string `json:"description,omitempty"`
+
+	Revision *int64 `json:"revision,omitempty"`
+}
+
+type Configurations struct {
+	// A list of information about the configuration. Does not apply to RabbitMQ
+	// brokers.
+	Current *ConfigurationID `json:"current,omitempty"`
+
+	History []*ConfigurationID `json:"history,omitempty"`
+	// A list of information about the configuration. Does not apply to RabbitMQ
+	// brokers.
+	Pending *ConfigurationID `json:"pending,omitempty"`
+}
+
+type EncryptionOptions struct {
+	KMSKeyID *string `json:"kmsKeyID,omitempty"`
+
+	UseAWSOwnedKey *bool `json:"useAWSOwnedKey,omitempty"`
+}
+
+type EngineVersion struct {
+	Name *string `json:"name,omitempty"`
+}
+
+type LDAPServerMetadataInput struct {
+	Hosts []*string `json:"hosts,omitempty"`
+
+	RoleBase *string `json:"roleBase,omitempty"`
+
+	RoleName *string `json:"roleName,omitempty"`
+
+	RoleSearchMatching *string `json:"roleSearchMatching,omitempty"`
+
+	RoleSearchSubtree *bool `json:"roleSearchSubtree,omitempty"`
+
+	ServiceAccountPassword *string `json:"serviceAccountPassword,omitempty"`
+
+	ServiceAccountUsername *string `json:"serviceAccountUsername,omitempty"`
+
+	UserBase *string `json:"userBase,omitempty"`
+
+	UserRoleName *string `json:"userRoleName,omitempty"`
+
+	UserSearchMatching *string `json:"userSearchMatching,omitempty"`
+
+	UserSearchSubtree *bool `json:"userSearchSubtree,omitempty"`
+}
+
+type LDAPServerMetadataOutput struct {
+	Hosts []*string `json:"hosts,omitempty"`
+
+	RoleBase *string `json:"roleBase,omitempty"`
+
+	RoleName *string `json:"roleName,omitempty"`
+
+	RoleSearchMatching *string `json:"roleSearchMatching,omitempty"`
+
+	RoleSearchSubtree *bool `json:"roleSearchSubtree,omitempty"`
+
+	ServiceAccountUsername *string `json:"serviceAccountUsername,omitempty"`
+
+	UserBase *string `json:"userBase,omitempty"`
+
+	UserRoleName *string `json:"userRoleName,omitempty"`
+
+	UserSearchMatching *string `json:"userSearchMatching,omitempty"`
+
+	UserSearchSubtree *bool `json:"userSearchSubtree,omitempty"`
+}
+
+type Logs struct {
+	Audit *bool `json:"audit,omitempty"`
+
+	General *bool `json:"general,omitempty"`
+}
+
+type LogsSummary struct {
+	Audit *bool `json:"audit,omitempty"`
+
+	AuditLogGroup *string `json:"auditLogGroup,omitempty"`
+
+	General *bool `json:"general,omitempty"`
+
+	GeneralLogGroup *string `json:"generalLogGroup,omitempty"`
+	// The list of information about logs to be enabled for the specified broker.
+	Pending *PendingLogs `json:"pending,omitempty"`
+}
+
+type PendingLogs struct {
+	Audit *bool `json:"audit,omitempty"`
+
+	General *bool `json:"general,omitempty"`
+}
+
+type SanitizationWarning struct {
+	AttributeName *string `json:"attributeName,omitempty"`
+
+	ElementName *string `json:"elementName,omitempty"`
+}
+
+type User struct {
```

Comment:
do we want to rename the User struct as User_SDK? we didn't change anything on this struct it is autogenerated

Reply:
@praveenghuge I see. When I remove `User` from the ignored resources, this will get autoamtically renamed by the code generator. We can ignore this comment.